autoload/neomake.vim

```diff
@@ -63,6 +63,9 @@ function! neomake#MakeJob(maker) abort
         let jobinfo.id = make_id
         " Assign this before neomake#MakeHandler gets run synchronously
         let s:jobs[make_id] = jobinfo
+        if has_key(a:maker, 'exit_callback')
```

Comment:
Since the `maker` is attached to `jobinfo`, there is no need to add `exit_callback` to `jobinfo`. Just get `exit_callback` from `jobinfo.maker` when you're ready to call it.